tap_mailchimp/schemas/list_members.json

```diff
@@ -210,6 +269,63 @@
         "null",
         "string"
       ]
+    },
+    "marketing_permissions": {
+      "type": [
+        "object"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "marketing_permission_id": {
+          "type": [
+            "string"
+          ]
+        },
+        "text": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "enabled": {
+          "type": [
+            "null",
+            "boolean"
+          ]
+        }
+      }
+    },
+    "last_note": {
```

Comment:
Will `last_note` be a dictionary every time?

Reply:
In the sample API response, it’s mentioned as a dictionary. Ref: https://mailchimp.com/developer/marketing/api/list-members/list-members-info/